CRM/Contribute/BAO/ContributionRecur.php

```diff
@@ -897,11 +897,15 @@ protected static function isComplete($recurringContributionID, $installments) {
    * @return array
    */
   public static function calculateRecurLineItems($recurId, $total_amount, $financial_type_id) {
-    $originalContributionID = CRM_Core_DAO::getFieldValue('CRM_Contribute_DAO_Contribution', $recurId, 'id', 'contribution_recur_id');
-    $lineItems = CRM_Price_BAO_LineItem::getLineItemsByContributionID($originalContributionID);
+    $originalContribution = civicrm_api3('Contribution', 'getsingle', array('contribution_recur_id' => $recurId));
```

Comment:
@Elennio  - this is the line where I think the missing 'is_test' => '' is causing the test fal